content/doc/developer/handling-requests/json.adoc

```diff
@@ -0,0 +1,255 @@
+---
+title: Expose HTTP API with json content with Jenkins
+summary: How to write GET and POST HTTP API with Jenkins, and how to test it.
+layout: developersection
+references:
+- url: https://github.com/stapler/stapler/blob/master/docs/reference.adoc
+  title: Stapler URL Binding Reference
+- url: ../routing
+  title: How requests in Jenkins are routed
+- url: ../../testing
+  title: Testing Jenkins
+---
+
+== Expose HTTP API with json content with Jenkins
+
+It's a common requirement for automation to expose Json object over HTTP API in server. This page show how a plugin can create a Jenkins Extension to expose HTTP API for GET and POST request, and how to test it with JenkinsRules (from acceptance-test-harness).
+
+=== Pre-requisite : a POJO (Plain Old Java Object)
+
+For this example we are using a very simple Object.
```

Comment:
Again, for the new-be trying to understand, what is the POJO for? What is its context ?